.github/workflows/deploy-android-to-play-store-azure.yml

```
@@ -0,0 +1,101 @@
+    name: android build
+    runs-on: ubuntu-22.04
+    if: inputs.android == true
+    defaults:
+      run:
+        working-directory: src/mobile
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)